PyInstaller/hooks/hook-gi.repository.GdkPixbuf.py

```diff
@@ -26,15 +26,18 @@
 destpath = "lib/gdk-pixbuf-2.0/2.10.0/loaders"
 cachedest = "lib/gdk-pixbuf-2.0/2.10.0"
 
-# If the "gdk-pixbuf-query-loaders" command is not in the current ${PATH}, GDK
-# and thus GdkPixbuf is unavailable. Return with a non-fatal warning.
+# If the "gdk-pixbuf-query-loaders" command is not in the current ${PATH}, or
+# is not in the GI lib path, GDK and thus GdkPixbuf is unavailable. Return with
+# a non-fatal warning.
 gdk_pixbuf_query_loaders = None
 
-if compat.architecture == '64bit':
-    # CentOS/Fedora package as -64
-    cmds = ['gdk-pixbuf-query-loaders-64', 'gdk-pixbuf-query-loaders']
-else:
-    cmds = ['gdk-pixbuf-query-loaders']
+libdir = get_gi_libdir('GdkPixbuf', '2.0')
```

Comment:
This will [throw a ValueError](https://github.com/pyinstaller/pyinstaller/blob/924f461a5dc87af05a8a3ff2d1618ddea5f65981/PyInstaller/utils/hooks/gi.py#L45) if `GdkPixbuf` isn't installed, which is (a) probably not the behavior we want here and (b) contrary to the comment above.

Reply:
@virtuald Thanks for the feedback and backup on that one. I'll work on coming up with a better solution, maybe a try/except would be appropriate.